islands/LiveControls.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from "preact/hooks";
+import { Page, Site } from "$live/types.ts";
+import InspectVSCode from "inspect_vscode/island.tsx";
+
+interface Props {
+  site: Site;
+  page: Page;
+  isProduction: boolean;
+}
+
+type IframeCommand = {
+  type: "scrollToComponent";
+  args: { id: string };
+};
+
+export const sendCommandToIframe = ({
+  iframe,
+  command,
+  targetOrigin,
+}: {
+  iframe: HTMLIFrameElement;
+  command: IframeCommand;
+  targetOrigin: string;
+}) => {
+  iframe.contentWindow?.postMessage(command, targetOrigin);
+};
+
+export default function LiveControls({ site, page, isProduction }: Props) {
+  const [inspectActive, setInspectActive] = useState(false);
+  const handleInspectClick = (event: MouseEvent) => {
+    if (event.defaultPrevented) {
+      return;
+    }
+
+    event.stopPropagation();
+    if (window.inspectVSCode) {
+      if (window.inspectVSCode.isActive()) {
+        window.inspectVSCode.deactivate();
+        setInspectActive(false);
+      } else {
+        window.inspectVSCode.activate(() => setInspectActive(false));
```

Comment:
Does this setInspectActive(false) from line 39 required, knowing the InspectVSCode will call the callback passed on line 41?